tests/process_monitor/test_process_monitor.py

```diff
@@ -566,10 +708,10 @@ def test_MantarrayProcessesMonitor__hard_stops_and_joins_processes_and_logs_queu
 
     actual = mocked_logger.call_args_list[1][0][0]
     assert "Remaining items in process queues: {" in actual
-    assert expected_ok_comm_item in actual
-    assert expected_file_writer_item in actual
-    assert expected_da_item in actual
-    assert expected_server_item in actual
+    assert str(expected_file_writer_item) in actual
```

Comment:
Think the redactions are made in the expected item here either

Reply:
I'm confused about what function to call to complete the redaction